src/scancode/cli.py

```diff
@@ -231,11 +231,51 @@ class ScanCommand(BaseCommand):
     short_usage_help = '''
 Try 'scancode --help' for help on options and arguments.'''
 
-    def get_params(self, ctx):
+    def format_options(self, ctx, formatter):
         """
-        Add options returned by plugins to the params list
+        Write all options into the formatter in groups they belong to.
+        If a group is not defined, add the option to `misc` group
         """
-        return super(BaseCommand, self).get_params(ctx)
```

Comment:
No, that function is a leftover from when I was adding `add_cmdline_option`, anyway this method is properly overridden in #704